src/frontends/paddle/src/op/reverse.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/opsets/opset1.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+
+using namespace default_opset;
+
+NamedOutputs reverse(const NodeContext& node) {
+    auto x = node.get_input("X");
```

Comment:
Paddle reverse supports TensorArray. https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/fluid/layers/tensor/reverse_en.html 
We need to check if X is such type and throw exception as it is not supported.

Reply:
> Paddle reverse supports TensorArray. https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/fluid/layers/tensor/reverse_en.html We need to check if X is such type and throw exception as it is not supported.

have fixed it.

Reply:
> Paddle reverse supports TensorArray. https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/fluid/layers/tensor/reverse_en.html We need to check if X is such type and throw exception as it is not supported.

For now, its input just supports tensor instead of LoDTensor since ov loads tensor array as tensor.

Reply:
Do you have the check?

BTW please let the reviewers unresolve the conversion.

Reply:
- Yes. I have checked the input of pp-shitu model and am sure that the one input can meet the requirements.

- If the input is lod_tensor, reverse should reverse the input array and then reverse each element of the array. But it seems hard to create the lod_tensor test to cover the lod_tensor reverse codes.

- So, I just set an assertion here.

Reply:
ok, there is indeed an assertion here. but it is not this assert like get_input_size. It should be something to check the input tensor type is a tensor_Array. 
However that I checked there is no infrastructure so far here to let you check it. Still need more work.
So my suggestion is to remove this assert here for now. And I will add the check in another PR with more warnings in other places for tensorarray.

Reply:
Got it.
Maybe we can encapsulate the paddle native info into the openvino node data structure later.